qipr_approver/approver/models/audit_trail_models.py

```diff
@@ -4,7 +4,7 @@
 from approver.models import Person
 
 class AuditTrail(models.Model):
-    user = models.ForeignKey(User, related_name='audit', null=True)
+    user = models.ForeignKey(User,on_delete=models.CASCADE,related_name='audit', null=True)
```

Comment:
So I guess we need to specify how we resolve foreign key issues now? Not a bad thing to require